packages/suite/src/components/wallet/CoinjoinSummary/CoinjoinStatusWheel/ProgressContent.tsx

```diff
@@ -0,0 +1,159 @@
+import React from 'react';
+import styled, { css } from 'styled-components';
+import { FluidSpinner, Icon, useTheme, variables } from '@trezor/components';
+import { Translation } from '@suite-components/Translation';
+import { CountdownTimer } from '@suite-components/CountdownTimer';
+import { useSelector } from '@suite-hooks/useSelector';
+import {
+    selectCurrentCoinjoinWheelStates,
+    selectCurrentSessionDeadlineInfo,
+    selectRoundsDurationInHours,
+} from '@wallet-reducers/coinjoinReducer';
+import { useCoinjoinSessionBlockers } from '@suite/hooks/coinjoin/useCoinjoinSessionBlockers';
+
+export const Container = styled.div<{ isWide: boolean }>`
+    width: ${({ isWide }) => `calc(100% - ${isWide ? 12 : 8}px)`};
+    height: ${({ isWide }) => `calc(100% - ${isWide ? 12 : 8}px)`};
+    background: ${({ theme }) => theme.BG_WHITE};
+    border-radius: 50%;
+    transition: background 0.15s ease-out, width 0.15s ease-out, height 0.15s ease-out;
+`;
+
+const CenteringContainer = styled.div`
+    position: absolute;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    width: 80px;
+    height: 80px;
+    left: calc(50% - 40px);
+    top: calc(50% - 40px);
+`;
+const AllPrivateContent = styled.div`
+    padding-top: 2px;
+    color: ${({ theme }) => theme.TYPE_GREEN};
+
+    p {
```

Comment:
I would strongly prefer not to use nested styling and always define separate style/component if possible.

Reply:
5455abfaf